server/graphql/document/src/mutations/patient/update.rs

```diff
@@ -0,0 +1,80 @@
+use async_graphql::*;
+use graphql_core::{
+    standard_graphql_error::{validate_auth, StandardGraphqlError},
+    ContextExt,
+};
+use service::{
+    auth::{Resource, ResourceAccessRequest},
+    document::patient::{UpdatePatient, UpdatePatientError},
+};
+
+use crate::queries::PatientNode;
+
+#[derive(InputObject)]
+pub struct UpdatePatientInput {
+    /// Patient document data
+    pub data: serde_json::Value,
+    /// The schema id used for the patient data
+    pub schema_id: Option<String>,
```

Comment:
why would it be optional ?

Reply:
None, would mean to use the default patient schema. Currently there isn't a default schema in the table though (plan to change that). Will make a note to iterate on this.